src/support/ios.c

```diff
@@ -1083,6 +1083,18 @@ int ios_ungetc(int c, ios_t *s)
     return c;
 }
 
+int ios_seekback(ios_t *s, int pos) {
```

Comment:
This isn't necessary. `ios_skip` with a negative argument has a correct implementation of this.